inbc-program/inbc/parser.py

```diff
@@ -101,6 +101,8 @@ def parse_fota_args(self) -> None:
                                  type=lambda x: validate_string_less_than_n_characters(x, 'Username', 50))
         parser_fota.add_argument('--target', '-t', nargs='*',
                                  default=['None'], required=False, help=TARGETS_HELP)
+        parser_fota.add_argument('--guid', '-gu', required=False, help='Firmware guid update',
+                                 type=lambda x: validate_string_less_than_n_characters(x, 'guid', 50))
```

Comment:
What is the expected format of a guid? We should check that the string is a valid guid before passing it into XML.

Reply:
@gblewis1 good point.

Reply:
@Siva472 - To add a little bit to what Gavin is saying.  Since the GUID would only have letters and numbers and a dash in it, we should ensure no other characters other than those could be injected.